webservices/resources/candidates.py

```diff
@@ -32,6 +32,7 @@ class CandidateList(ApiResource):
     schema = schemas.CandidateSchema
     page_schema = schemas.CandidatePageSchema
     filter_multi_fields = filter_multi_fields(models.Candidate)
+    filter_fulltext_fields = [('q', models.CandidateSearch.fulltxt)]
```

Comment:
For implementation, check out `ApiResource` in `webservices/common/views.py`. This particular class variable is a list of tuples relating query parameters (in this case, "q") to tsvector columns for use in filtering. Basically, DRF-lite.